components/builder-protocol/protocols/sessionsrv.proto

```diff
@@ -8,6 +8,7 @@ enum OAuthProvider {
   AzureAD = 3;
   GitLab = 4;
   Okta = 5;
+  ActiveDirectory = 6;
```

Comment:
Are there any other flavors of AD we know of that this naming might conflict with or otherwise complicate?

Reply:
perhaps? I think it'll depend on your AD setup. I'd say since this is the first case it's okay and we can expand it from there

Reply:
Yep - this can be our generic non-azure AD provider